README.md

```diff
@@ -58,12 +60,32 @@ to `local` removes the implication that any particular revision is used when run
 ``` shell
 echo "local" > .application-version
 ```
+#### Python version
 
 It is preferable to use the version of Python locally that matches that
 used on deployment. This project has a `.python_version` file for this
 purpose.
 
-Upgrade pip and install dependencies:
+#### Pyenv
+
+It is recommended to install `pyenv` python version management tool to easily switch between python versions.
+To install `pyenv` use this command:
+```shell
+curl https://pyenv.run | bash
+```
+After the installation it should tell you to execute a command to add `pyenv` to path. It should look something like this:
+```shell
+export PYENV_ROOT="$HOME/.pyenv"
+
+command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
+
+eval "$(pyenv init -)"
+```
+Python versions can be changed with `pyenv local` or `pyenv global` command suffixed with desired version (e.g. 3.11.4). Different versions of python can be installed first with `pyenv install` command. Refer to the pyenv project Readme [here](https://github.com/pyenv/pyenv). To avoid confusion check current python version at any given time using `python --version` or `python3 --version` command.
```

Comment:
```suggestion
Python versions can be changed with the `pyenv local` or `pyenv global` commands suffixed with the desired version (e.g. 3.11.4). Different versions of Python can be installed first with the `pyenv install` command. Refer to the pyenv project Readme [here](https://github.com/pyenv/pyenv). To avoid confusion, check the current Python version at any given time using `python --version` or `python3 --version`.